app-emulation/lxd/lxd-0.16.ebuild

```diff
@@ -140,16 +142,32 @@ src_install() {
 }
 
 pkg_config() {
+
+	# Installing a bridge is a convenience for the user, but the ebuild
+	# should succeed even if this fails.  Hence all the error checking
+	# before an unconditional 0 return code.
 	if use daemon; then
-		if brctl show lxcbr0 2>&1 | grep "No such device" >/dev/null; then
-			brctl addbr lxcbr0
+		showoutput=$(brctl show lxcbr0 2>&1)
+		if test ${?} -eq 0 && test -n "${showoutput}"; then
+			if echo ${showoutput} | grep "No such device" >/dev/null; then
```

Comment:
Would finding the index of this and testing on that be better than using echo and grep?  echo and grep seems out of place here.